gradle.properties

```diff
@@ -42,7 +42,7 @@ mockitoCoreVersion=2.28.2
 
 reactiveStreamsVersion=1.0.3
 jcToolsVersion=3.0.0
-jacksonVersion=2.10.3
+jacksonVersion=2.10.4
```

Comment:
The latest is 2.11.0, do you prefer to hold upgrading to it?

Reply:
for now lets stay on 2.10, but we should move to 2.11 soon (https://github.com/FasterXML/jackson/wiki/Jackson-Releases)